### ISSUE TITLE
Job submit script

### ISSUE THREAD

**Issue by oskarvid** [neicnordic/Bifrost#17]
The job submit script has been created now, it needs to be edited with all of the schizophrenia related code. Currently it has some placeholders.

**Comment by oskarvid** [comment]
Added file transfer field creation in this commit: https://github.com/neicnordic/Bifrost/commit/2509a22d6a834ed351ed29b48c1aa364aefeb0bb

**Comment by oskarvid** [comment]
The [latest commit](https://github.com/neicnordic/Bifrost/commit/b4600d907a8dac29c5e56f55c3d1aa2a02d4fe15) added encryption of the input file by the submit script, and right now it contains scp code so we can simulate job submission to a server outside TSD. And there are hardcoded paths that need to be generalized.

**Comment by oskarvid** [comment]
The [latest commit](https://github.com/neicnordic/Bifrost/commit/2f4fd907f01735044aecbe57c9d320c693a5eacf) added the following:  
* Absolute paths for the paths given as argument to the submit script
* Encrypted files, and the new config file, get put in a new directory instead of straight into the Bifrost directory
  * Now this directory is transferred instead of just the input file and config file, this improves the receiving side handling of new transfers
* The encrypt command line has been generalized, no more hard coded paths

**Comment by oskarvid** [comment]
In the [latest commit](https://github.com/neicnordic/Bifrost/commit/16f16fec2a87a426b59d84059d3193402877a842#diff-38aaeadbff72acbc47d397996d391950) I overhauled the imputation related part of the script.  
* Since the input file comes already encrypted the encryption part has been removed.
* I have moved away from testing the submit function on the outside, i.e I removed the scp related code and added the TSD S3 api file transfer function instead. This is currently hardcoded for imputation jobs until the schizophrenia code also moves away from using scp for local testing purposes.
